src/librustc_codegen_llvm/debuginfo/metadata.rs

```diff
@@ -1504,7 +1507,8 @@ impl VariantMemberDescriptionFactory<'ll, 'tcx> {
 
 #[derive(Copy, Clone)]
 enum EnumDiscriminantInfo<'ll> {
-    RegularDiscriminant(&'ll DIType),
+    /// First field is the field index of the discriminant.
```

Comment:
Please make this a struct variant with the documentation on the fields and not the variant

Reply:
Done